distribution/graal/skywalking-generate-plugin/pom.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.skywalking</groupId>
+        <artifactId>graal</artifactId>
+        <version>9.7.0-SNAPSHOT</version>
+    </parent>
+
+    <packaging>maven-plugin</packaging>
+
+    <artifactId>skywalking-generate-plugin</artifactId>
```

Comment:
This should be renamed to `oal-on-graal-compiling-plugin`. And all existing codes in this are not runtime relative, right?

Reply:
Yes, but not only oal code, also envoy-metrics.

Reply:
What do you mean envoy-metrics? I think you misuse terms. In the SkyWalking OAP, metrics are only generated by OAL or MAL, regardless of the original data sources.

Reply:
Sorry,
```
            FieldsHelper.SINGLETON.init("metadata-service-mapping.yaml", ServiceMetaInfo.class);
```
is also executed by this plugin.

Reply:
That is Envoy ALS relative only. Please separate them in the right place. Same as suggested here, https://github.com/apache/skywalking/pull/11354#discussion_r1335581945

Reply:
I think perhaps it shouldn't be changed, because there might be more things that need to be generated during construction in the future, and we shouldn't create many plugins to generate different content.

Reply:
This has not relationship with the OAL. So it should be in two places.